charts/nfspvc-operator/values.yaml

```diff
@@ -11,14 +11,6 @@ image:
     # -- The pull policy for the image.
     pullPolicy: IfNotPresent
 
-  kubeRbacProxy:
-    # -- The repository of the kube-rbac-proxy container image.
```

Comment:
remove also the kubeRbacProxy section that configures the kube-rbac-proxy container